docs/docs/cmd/spo/tenant/tenant-commandset-add.md

```diff
@@ -0,0 +1,65 @@
+# spo tenant commandset add
+
+Add a ListView Command Set as a tenant wide extension.
+
+## Usage
+
+```sh
+m365 spo tenant commandset add [options]
+```
+
+## Options
+
+`-t, --title <title>`
+: The title of the ListView Command Set.
+
+`-l, --listType <listType>`
+: The list or library type to register the ListView Command Set on. Allowed values `List` or `Library`.
+
+`-i, --clientSideComponentId <clientSideComponentId>`
+: The Client Side Component Id (GUID) of the ListView Command Set.
+
+`-p, --clientSideComponentProperties [clientSideComponentProperties]`
+: The Client Side Component properties of the ListView Command Set.
+
+`-w, --webTemplate [webTemplate]`
+: Optionally add a web template (e.g. STS#3, SITEPAGEPUBLISHING#0, etc) as a filter for what kind of sites the ListView Command Set is registered on.
+
+`--location [location]`
+: The location of the ListView Command Set. Allowed values `ContextMenu`, `CommandBar` or `Both`. Defaults to `CommandBar`.
+
+--8<-- "docs/cmd/_global.md"
+
+## Remarks
+
+Running this command from the Windows Command Shell (cmd.exe) or PowerShell for Windows OS XP, 7, 8, 8.1 without bash installed might require additional formatting for command options that have JSON, XML or JavaScript values because the command shell treat quotes differently. For example, this is how commandset user custom action can be created from the Windows cmd.exe:
+
+```sh
+m365 spo tenant commandset add --title "YourAppCustomizer" --clientSideComponentId b41916e7-e69d-467f-b37f-ff8ecf8f99f2 --clientSideComponentProperties '{\"testMessage\":\"Test message\"}'
+```
+
+Note, how the clientSideComponentProperties option has escaped double quotes `'{\"testMessage\":\"Test message\"}'` compared to execution from bash `'{"testMessage":"Test message"}'`.
+
+!!! warning "Escaping JSON in PowerShell"
+    When using the `--clientSideComponentProperties` option it's possible to enter a JSON string. In PowerShell 5 to 7.2 [specific escaping rules](./../../../user-guide/using-cli.md#escaping-double-quotes-in-powershell) apply due to an issue. Remember that you can also use [file tokens](./../../../user-guide/using-cli.md#passing-complex-content-into-cli-options) instead.
+
+!!! important
+    To use this command, you need to be a SharePoint Admin.
+
+## Examples
+
+Adds a ListView Command Set that's deployed tenant wide to CommandBars of Lists.
+
+```sh
+m365 spo tenant commandset add --title "Some customizer" --clientSideComponentId  799883f5-7962-4384-a10a-105adaec6ffc --listType List
+```
+
+Adds a ListView Command Set that is configured to the context menu of communication site libraries.
+
+```sh
+m365 spo tenant commandset  add --title "Some customizer" --clientSideComponentId  799883f5-7962-4384-a10a-105adaec6ffc --webTemplate "SITEPAGEPUBLISHING#0" --listType Library --location ContextMenu
+```
+
+## Response
+
+The command won't return a response on success.
```

Comment:
I think that is not true. When I played a bit with the command I get a response 
![image](https://github.com/pnp/cli-microsoft365/assets/58668583/eb333e57-3ca2-4810-b232-1a3832622256)

I think it's because we use 
```
await Cli.executeCommand(spoListItemAddCommand as Command, { options: { ...commandOptions, _: [] } })
``` 
If we don't want the output I suppose we should use `executeCommandWithOutput` and not log the output

Reply:
I overlooked this one apparently 😂😂

Reply:
ok so let's set it to draft until this is done 😉